test/products-route-tests.spec.js

```diff
@@ -0,0 +1,57 @@
+const session = require('supertest-session');
+const { app } = require('../server/index');
+const expect = require('chai').expect;
+
+const product = {
+    name: 'Lettuce',
+    price: 2.00,
+    description: 'Very fresh green romaine lettuce!'
+}
+
+describe('/api/products routes', () => {
+    describe('/products GET URI', () => {
+        it('should send a valid JSON response', () => {
+            app
+                .get('/products')
+                .expect(200)
+                .expect('Content-Type', /json/)
```

Comment:
Lets aim for more specificity. Check the actual contents of the reply - getting a json 200 doesnt mean you got the right json.